tensorflow/java/src/gen/perl/tftypes-runall.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
```

Comment:
My recommendation would be to do away with this script and instead have `tftypes.pl` just generate all the files, always. No flags, no options - it generates all files in place. Thoughts?